packages/frontend/src/hooks/useBigNumberConfig.ts

```diff
@@ -0,0 +1,58 @@
+import {
+    ApiQueryResults,
+    BigNumber,
+    Explore,
+    findFieldByIdInExplore,
+    friendlyName,
+    getFieldLabel,
+} from 'common';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+
+const useBigNumberConfig = (
+    bigNumberConfigData: BigNumber | undefined,
+    resultsData: ApiQueryResults | undefined,
+    explore: Explore | undefined,
+) => {
+    const metric = resultsData?.metricQuery.metrics[0];
```

Comment:
@nathaliabj if there are no metrics available we should take the first dimension instead. 

Later this can be configurable too !

Reply:
@owlas In this case though we should only take metrics as dimensions more often than not are not numbers ? or am i mistaken on that?

Reply:
is it a problem if it's a string?